pandas/core/generic.py

```diff
@@ -1940,6 +1940,79 @@ def tail(self, n=5):
             return self
         return self.iloc[-n:]
 
+  
+    def sample(self, n=None, frac=None , replace=False, weights=None, seed=None):
+        """
+        Returns a sample of rows from object. 
+        
+        Parameters
+        ----------
+        n: in, optional, Default = 5 if frac = None. 
```

Comment:
this should be `int` not `in`.

Also, the numpy docstring convention is usually *not* to list default values after the type. Rather, they should be described in the description part below.